### PR TITLE
Topic 0 exercise 2

### PR DESCRIPTION
**Pull request by MarcosBrian**
here topic 0 exercise 2 resolved

### CODE REVIEW

File: TOPIC-0-EXERCISE-1/exercise 1-iii.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+</head>
+<body>
+	<h2>My daily chores</h2>
+		<ul>
+  			<li>Make bed</li>
+  			<li>Prepare breakfast</li>
+  			<li>Work</li>
+  			<li>Prepare lunch</li>
+  			<li>Program</li>
+        <li>Work</li>
```
Comment (DemianMoschin-GLB) [inline TOPIC-0-EXERCISE-1/exercise 1-iii.html]: Indent

File: TOPIC-0-EXERCISE-1/exercise 1-vi.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+				<td>E-Mail:</td>
+				<td><input type="email" name="email" required></td>
+			</tr>
+			<tr>
+				<td  text-align="center">Birthday:</td>
+				<td><input type="date" name="date" required></td>
+			</tr>
+			<tr>
+				<td>Your Favourite Sport:</td>
+				<td><select required="required">
```
Comment (DemianMoschin-GLB) [inline TOPIC-0-EXERCISE-1/exercise 1-vi.html]: please avoid joining tags, use indentation and separate lines

File: TOPIC-0-EXERCISE-2/exercise 2-1/exercise 2-1.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<title></title>
+		<meta name="description" content="">
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+		<link rel="manifest" href="site.webmanifest">
+		<link rel="apple-touch-icon" href="icon.png">
+		<!-- Place favicon.ico in the root directory -->
```
Comment (DemianMoschin-GLB) [inline TOPIC-0-EXERCISE-2/exercise 2-1/exercise 2-1.html]: ?

File: TOPIC-0-EXERCISE-2/exercise 2-1/exercise 2-1.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+		</footer>   
+	</section>
+	<aside>
+		ASIDE
+	</aside>
+	<footer>
+		FOOTER
+	</footer>
+		
+		<script src="js/vendor/modernizr-{{MODERNIZR_VERSION}}.min.js"></script>
```
Comment (DemianMoschin-GLB) [inline TOPIC-0-EXERCISE-2/exercise 2-1/exercise 2-1.html]: Please review this: https://developers.google.com/speed/docs/insights/BlockingJS

File: TOPIC-0-EXERCISE-2/exercise 2-3/exercise 2-3.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+		<article><p>SECTION ARTICLE</p></article>
+		<footer><p>SECTION FOOTER</p></footer>   
+	</section>
+	<aside class="sidebar">
+		<p>ASIDE</p>
+	</aside>
+	<footer class="footer">
+		<p>FOOTER</p>
+	</footer> 
+		<script src="js/vendor/modernizr-{{MODERNIZR_VERSION}}.min.js"></script>
```
Comment (DemianMoschin-GLB) [inline TOPIC-0-EXERCISE-2/exercise 2-3/exercise 2-3.html]: Same as  above

File: TOPIC-0-EXERCISE-1/exercise 1-vi.html (hunk 2 above)
Comment (DemianMoschin-GLB) [inline TOPIC-0-EXERCISE-1/exercise 1-vi.html]: Please review the ```required``` attribute

File: TOPIC-0-EXERCISE-1/exercise 1-iii.html (hunk 1 above)
Comment (MarcosBrian) [reply TOPIC-0-EXERCISE-1/exercise 1-iii.html]: done

File: TOPIC-0-EXERCISE-2/exercise 2-1/exercise 2-1.html (hunk 4 above)
Comment (MarcosBrian) [reply TOPIC-0-EXERCISE-2/exercise 2-1/exercise 2-1.html]: reviewed

File: TOPIC-0-EXERCISE-2/exercise 2-5-1/exercise 2-5-1.html [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+	</footer>
+		
+		<script src="js/vendor/modernizr-{{MODERNIZR_VERSION}}.min.js"></script>
+		<script src="https://code.jquery.com/jquery-{{JQUERY_VERSION}}.min.js" integrity="{{JQUERY_SRI_HASH}}" crossorigin="anonymous"></script>
+		<script>window.jQuery || document.write('<script src="js/vendor/jquery-{{JQUERY_VERSION}}.min.js"><\/script>')</script>
+		<script src="js/plugins.js"></script>
+		<script src="js/main.js"></script>
+
+		<!-- Google Analytics: change UA-XXXXX-Y to be your site's ID. -->
```
[the remark this answers is not in the document]
Comment (MarcosBrian) [reply TOPIC-0-EXERCISE-2/exercise 2-5-1/exercise 2-5-1.html]: done

File: TOPIC-0-EXERCISE-1/exercise 1-vi.html (hunk 2 above)
Comment (MarcosBrian) [reply TOPIC-0-EXERCISE-1/exercise 1-vi.html]: done

File: TOPIC-0-EXERCISE-2/exercise 2-1/exercise 2-1.html (hunk 3 above)
Comment (MarcosBrian) [reply TOPIC-0-EXERCISE-2/exercise 2-1/exercise 2-1.html]: deleted

File: TOPIC-0-EXERCISE-1/exercise 1-vi.html (hunk 2 above)
Comment (DemianMoschin-GLB) [reply TOPIC-0-EXERCISE-1/exercise 1-vi.html]: still needed as it is?

File: TOPIC-0-EXERCISE-1/exercise 1-vi.html [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+			</tr>
+			<tr>
+				<td><button type="submit" value="Submit">Submit</button></td>
+				<td><button type="reset" value="Reset">Reset</button></td>
+			</tr>
+
+		</table>
+	</form>	
+</body>
+</html>
```
Comment (DemianMoschin-GLB) [inline TOPIC-0-EXERCISE-1/exercise 1-vi.html]: Add new line

File: TOPIC-0-EXERCISE-2/exercise 2-1/exercise 2-1.html [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+		<script src="https://code.jquery.com/jquery-{{JQUERY_VERSION}}.min.js" integrity="{{JQUERY_SRI_HASH}}" crossorigin="anonymous"></script>
```
Comment (DemianMoschin-GLB) [inline TOPIC-0-EXERCISE-2/exercise 2-1/exercise 2-1.html]: Please review this: 
Where did you declared this variables?
Are you sure that you are using this script as it should?

File: TOPIC-0-EXERCISE-2/exercise 2-1/exercise 2-1.html (hunk 4 above)
Comment (DemianMoschin-GLB) [reply TOPIC-0-EXERCISE-2/exercise 2-1/exercise 2-1.html]: Please review comment below

File: TOPIC-0-EXERCISE-2/exercise 2-2/exercise 2-2.html [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+		</footer>   
+	</section>
+	<aside>
+		ASIDE
+	</aside>
+	<footer>
+		FOOTER
+	</footer>
+		
+		<script src="js/vendor/modernizr-{{MODERNIZR_VERSION}}.min.js"></script>
```
Comment (DemianMoschin-GLB) [inline TOPIC-0-EXERCISE-2/exercise 2-2/exercise 2-2.html]: Please see all Comments above

File: TOPIC-0-EXERCISE-2/exercise 2-3/exercise 2-3.html [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+		<link rel="stylesheet" href="css/normalize.css">
+		<link rel="stylesheet" href="css/main.css">
+		<link href="style.css" rel="stylesheet">
+		<script src="modernizr-custom.min.js"></script>
+	</head>
+
+	<body>
+		<header class="header">
+			<p>
+				HEADER
```
Comment (DemianMoschin-GLB) [inline TOPIC-0-EXERCISE-2/exercise 2-3/exercise 2-3.html]: Review indent for all <p> content

File: TOPIC-0-EXERCISE-2/exercise 2-3/exercise 2-3.html [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+			</p>
+		</aside>
+		
+		<footer class="footer">
+			<p>
+				FOOTER
+			</p>
+		</footer> 
+		
+		<script src="js/vendor/modernizr-{{MODERNIZR_VERSION}}.min.js"></script>
```
Comment (DemianMoschin-GLB) [inline TOPIC-0-EXERCISE-2/exercise 2-3/exercise 2-3.html]: Same as above

File: TOPIC-0-EXERCISE-2/exercise 2-3/exercise 2-3.html [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+		<script src="https://code.jquery.com/jquery-{{JQUERY_VERSION}}.min.js" integrity="{{JQUERY_SRI_HASH}}" crossorigin="anonymous"></script>
+		<script>window.jQuery || document.write('<script src="js/vendor/jquery-{{JQUERY_VERSION}}.min.js"><\/script>')</script>
+		<script src="js/plugins.js"></script>
+		<script src="js/main.js"></script>
+
+		<script>
+			window.ga=function(){ga.q.push(arguments)};ga.q=[];ga.l=+new Date;
+			ga('create','UA-XXXXX-Y','auto');ga('send','pageview')
+		</script>
+		<script src="https://www.google-analytics.com/analytics.js" async defer></script>
```
Comment (DemianMoschin-GLB) [inline TOPIC-0-EXERCISE-2/exercise 2-3/exercise 2-3.html]: Do you really need this?

File: TOPIC-0-EXERCISE-1/exercise 1-iii.html (hunk 1 above)
Comment (MarcosBrian) [reply TOPIC-0-EXERCISE-1/exercise 1-iii.html]: done

**Pull request merged**